code/__DEFINES/traits.dm

```diff
@@ -117,6 +117,10 @@
 			: FALSE)\
 		: FALSE)
 #define HAS_TRAIT_NOT_FROM(target, trait, source) (target.status_traits ? (target.status_traits[trait] ? (length(target.status_traits[trait] - source) > 0) : FALSE) : FALSE)
+/// Adds a trait, with the source being the caller's type.
+#define ADD_TYPED_TRAIT(target, trait)	ADD_TRAIT(target, trait, "[type]")
```

Comment:
Didn't another PR add this define? Same review applies from that to this in that variables are pulled from a non-explicit source

Reply:
Ah yeah, holopara PR did this, I forgot that this PR did too